packages/contracts-bedrock/hardhat.config.ts

```diff
@@ -38,6 +38,7 @@ const config: HardhatUserConfig = {
       chainId: Number(process.env.CHAIN_ID),
       url: process.env.L1_RPC || '',
       accounts: [process.env.PRIVATE_KEY_DEPLOYER || ethers.constants.HashZero],
+      live: process.env.VERIFY_CONTRACTS === 'true',
     },
```

Comment:
I see that when live is false, it skips verification here: https://github.com/ethereum-optimism/optimism/blob/ea2cf94cc5940f9b91bc976a2095a167c5a61b61/packages/contracts-bedrock/src/deploy-utils.ts#L77